LABTASKS.md

```diff
@@ -20,8 +20,13 @@ what the answer to a question is, etc.
 The client side of our project has changed since lab #2. The directory structure is
 mostly the same, but the client-side interface uses Angular to handle most of the creation
 of the elements of the user interface. Angular's template syntax extends HTML and JavaScript.
-The testing is handled in two new places (Angular spec files for testing Angular
-and E2E tests) with new tools (Jasmine/Karma and Cypress).
+The testing is handled in two new places:
+
+* Angular spec files (e.g., `user-list.component.spec.ts`) for unit
+  testing in Angular. These use Jasmine and Karma.
+* E2E (end-to-end) tests in `client/e2e`. These use Cypress.
+
+![Location of testing code](https://user-images.githubusercontent.com/302297/108024936-25605500-6feb-11eb-87e5-829d4e9de44a.png)
```

Comment:
I'm not sure where this image comes from but the icons and theme here looks very unlike what most students will actually have.

Reply:
Thanks! That was just from my VS Code, but I think I have an extension that adds "fancier" icons to the explorer. I'll update it to use your image.

Reply:
Oh, foo. I'd need to re-do all the editing I did on it and I don't have time for that. We'll end up going with my image. :cry: